hutch_python/utils.py

```diff
@@ -433,3 +435,22 @@ def get_fully_qualified_domain_name():
         )
         logger.debug("getfqdn failed", exc_info=True)
         return ""
+
+
+abort_msg = """
+Your RunEngine has been interrupted.  If you aborted the current run (Ctrl+C),
+the RunEngine is ready for use.  If you paused the run, you must resume,
```

Comment:
We don't allow for pausing at the moment, so maybe we shouldn't mention it just yet?

Reply:
Very true, this was me being hopeful that I'd figure out the Ctrl+P type of functionality early on.  I'll see if I can't make any headway, otherwise I'll change the message